src/styles/_header.scss

```diff
@@ -37,5 +37,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
+    transition-timing-function: linear;
```

Comment:
When you use transform add transition-property
https://developer.mozilla.org/en-US/docs/Web/CSS/transition-property